src/main/java/friday/model/TaskList.java

```diff
@@ -0,0 +1,63 @@
+package friday.model;
+
+import friday.ui.Ui;
+import friday.storage.Storage;
+import friday.exception.FridayException;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Consists of all methods that modify the task list.
+ */
+public class TaskList {
+    private final List<Task> tasks = new ArrayList<>();
+
+    public List<Task> all() {
```

Comment:
Should this method be renamed to contain a verb?
```suggestion
    public List<Task> getAll() {
```